src/style.scss

```diff
@@ -8,9 +9,6 @@ a:hover {
   color: inherit;
   text-decoration: none;
 }
-.bgBlack {
-  background-color: #0e1414;
-}
 .fontSize16 {
```

Comment:
Again 😆

Does any bootstrap utility fit your need?

Reply:
Do you mean `fontSize16`?
Don't need it anymore since default `font-size` is 16px